src/lib/chips/chip-list.ts

```diff
@@ -317,4 +506,138 @@ export class MdChipList implements AfterContentInit, OnDestroy {
 
     return false;
   }
+
+  _setSelectionByValue(value: any, isUserInput: boolean = true) {
+    this._clearSelection();
+    this.chips.forEach((chip) => chip.deselect());
```

Comment:
Nit: don't need parenthesis around `chip`
(in any arrow function where the type is implicit)